hoodie-client/src/main/java/com/uber/hoodie/index/bloom/HoodieBloomIndex.java

```diff
@@ -38,9 +38,8 @@
 import com.uber.hoodie.exception.MetadataNotFoundException;
 import com.uber.hoodie.index.HoodieIndex;
 import com.uber.hoodie.table.HoodieTable;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
+
+import java.util.*;
```

Comment:
Can we please just include the classes we need ?

Reply:
I usually prefer the pattern where in, we have more than 3 entries imported from same package, will use wildcard, if not, will go with separate imports. Let me know if you want to have all as separate imports.